TicTacToeApp/Board.swift

```diff
@@ -0,0 +1,48 @@
+
+public class Board {
+  
+  private let markedSpaces: [Int: PlayerMark]
+  private let size = 3
+
+  public init() {
+    markedSpaces = [:]
+  }
+
+  public init(markedSpaces: [Int : PlayerMark]) {
+    self.markedSpaces = markedSpaces
+  }
+
+  public func markSpace(id: Int, mark: PlayerMark) -> Board {
```

Comment:
Yeah, that's a good point. We're changing it to space. Renaming methods that mention "spaceId" as well.